libPhoneNumberTests/NBPhoneNumberUtil+ShortNumberTestHelper.m

```diff
@@ -0,0 +1,53 @@
+//
+//  NBPhoneNumberUtil+ShortNumberTestHelper.m
+//  libPhoneNumber
+//
+//  Created by Paween Itthipalkul on 12/1/17.
+//  Copyright © 2017 Google LLC. All rights reserved.
+//
+
+#import "NBPhoneNumberUtil+ShortNumberTestHelper.h"
+
+#import "NBMetadataHelper.h"
+#import "NBPhoneMetadata.h"
+#import "NBPhoneNumberDesc.h"
+#import "NBPhoneNumberUtil+Category.h"
+
+#if SHORT_NUMBER_SUPPORT
+
+@implementation NBPhoneNumberUtil(ShortNumberTestHelper)
+
+- (NSString *)exampleShortNumberForCost:(NBEShortNumberCost)cost regionCode:(NSString *)regionCode {
+  NBPhoneMetaData *metadata = [self.helper shortNumberMetadataForRegion:regionCode];
+  if (metadata == nil) {
+    return @"";
```

Comment:
is returning an empty string better than returning nil?

Reply:
This is done to match the Java implementation of the same method.

I think matching the canonical implementation makes porting changes easier.

https://github.com/googlei18n/libphonenumber/blob/master/java/libphonenumber/src/com/google/i18n/phonenumbers/ShortNumberInfo.java#L375

Reply:
+1

Understand the oddities...but...I think it makes sense to try to match logic between the Java/Google version and this version...